app/editor/EditorContentCtrl.js

```diff
@@ -26,6 +23,42 @@
 
     init();
 
+    $rootScope.$on(EventService.EVENT.FILTER_TRANS_UNIT,
+      function (event, filter) {
+        if(filter.status.all === true) {
+          $location.search('status', null);
+        } else {
+          var queryString = '';
+
+          _.forEach(filter.status, function(val, key) {
+            if(val) {
+              queryString += key + ',';
+            }
+          });
+          queryString = queryString.substring(0, queryString.length - 1);
```

Comment:
This looks like it is stripping the last character from `queryString`. Is that what is intended? If so, what is the purpose of doing so?